ChangeLog

```diff
@@ -1,3 +1,116 @@
+2020-03-15, Version 5.11.0
+	* collectd: A Code of Conduct has been added. Thanks to octo. #3401
+	* text protocol: Support for meta_data in PUTVAL has been added. Thanks
+	  to RafaelMarinheiro, dago. #3381, #2726
+	* utils_cache: Internal API has changed - exposed meta_data_toc
+	  function. Thanks to jkohen. #3339
+	* filter_chain: Auto load matches and targets if AutoLoadPlugin option
+	  is set. Thanks to manuelluis. #1034
+	* AMQP plugin: TLS support has been added. Thanks to mbakhoff. #3391
+	* Battery plugin: Read from Linux sysfs charge files has been added.
+	  Thanks to maclomhair. #3190
+	* Buddyinfo plugin: New plugin for memory fragmentation info has been
+	  added. Thanks to asafkahlon. #3303
+	* capabilities plugin: New plugin to read static platform data has been
+	  added. Thanks to kwiatrox, dago. #3384, #3292
+	* DCPMM plugin: New plugin for Intel Optane DC Presistent Memory (DCPMM)
+	  has been added. Thanks to htg1. #3274
+	* DF plugin: LogOnce option has been added. Thanks to themylogin. #2910
+	* DPDK Telemetry plugin: New plugin to fetch DPDK metrics (with intent
+	  to replace dpdk_stat) has been added. Thanks to reshmapattan. #3273
+	* Disk plugin: A native (sysctl-based) implementation for NetBSD has
+	  been added. Thanks to efuss, dago. #3379, #3334
+	* GPU_Nvidia‏‎ plugin: Configurable plugin instance by GPU name
+	  and/or GPU index has been added. Thanks to rdietric. #3264
+	* Intel_PMU plugin: Support for uncore multi pmu has been added. Thanks
+	  to kwiatrox. #3388
+	* Ipstats plugin: New plugin to count incoming, outgoing, forwarded
+	  packets on FreeBSD has been added. Thanks to ometol. #3187
+	* Java plugin: Support boolean input has been added. Thanks to Caden94.
+	  #3044
+	* Logparser plugin: New plugin for filtering and parsing log messages
+	  has been added. Thanks to kwiatrox, krzysztofx, mmozejkx, aboczkox,
+	  rjablonx. #3045
+	* Network plugin: New metadata "network:ip_address" has been added.
+	  Thanks to ashie. #3191
+	* NFS plugin: An implementation for NetBSD (using sysctl) has been
+	  added. Thanks to efuss, dago. #3377, #3333
+	* NUT plugin: Support for the "output.realpower" value from the ups code
+	  has been added. Thanks to elratt0r. #2528
+	* Redfish plugin: New read plugin to collect out-of-band sensor data
+	  from Redfish endpoints has been added. Thanks to mmozejkx, aboczkox,
+	  mkobyli, kkepka, mansingh82, Zoltan Szabo. #2926
+	* Slurm plugin: The new SLURM plugin gathers metrics from the SLURM
+	  workload manager has been added. Thanks to pllopis. #3037
+	* Tail_CSV plugin: FieldSeparator option has been added. Thanks to
+	  redbaron4. #3394
+	* UBI plugin: The new "ubifs" plugin reports block state for flash
+	  memory devices with UBIFS filesystem has been added. Thanks to pticon.
+	  #3204
+	* Varnish plugin: Support for Varnish 6 has been added, deprecated for
+	  ver 2, 3 and 5. Thanks to ismaelpuerto. #3360
```

Comment:
Since this one has been reverted, have removed it from wiki, but probably was late (just few moments ago) and _after_ this patch has been made. Sorry for my procrastination, should fix this before weekend.